### PR TITLE
beato day 10 TLC 1 & 2

### PR DESCRIPTION
**Pull request by Avuxo**
I did these on monday but was too busy to get them in PR.

Shouldn't be so busy next week.

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_1_1.txt [hunk 1]
```diff
@@ -11,7 +11,7 @@ I chose a route that wouldn't attract attention, so it took a bit of time.
 [sha:c5fc530519a1350a43a2769d61e402c27cd33743]{
 -- Page 10001, Offset 24417.
 -- アルクェイドは……いた。
-Arcueid is... over there.
+Arcueid is... ah, over there.
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_1_1.txt:14]: maybe even 'ah, there she is.'

File: script/Arcueid/Day 10/10_00_ARC10_2_2.txt [hunk 2]
```diff
@@ -21,12 +21,12 @@ Isn't it because you wanted to take revenge on me?"
 [sha:fdd727e59a75764655e2643f661c55e44acd8c74]{
 -- Page 76, Offset 24448.
 -- ヴローヴの件で許してはもらったけど、あれはあくまで“元に戻せない”代わりの補填というか、借金は返せたけど借りていた時にかけた迷惑は残っているというか……」
-You may have forgiven me in Vlov's case, but that was only to make up for the fact that you couldn't return to normal, or that I may have paid my debt, but the issues I've caused during the borrowing period is still there..."
+You may have forgiven me for what happened while we fought Vlov, but that only happened while making up for the damage I caused in the first place. I may have paid off my debt, but the issues I've brought about during the borrowing period are still there..."
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_2_2.txt:24]: but that only made up for the damage I caused

File: script/Arcueid/Day 10/10_00_ARC10_2_2.txt [hunk 3]
```diff
@@ -171,12 +174,12 @@ Besides, what would you do if I told you that I hated you? Would you have let me
 [sha:f45fc28c827a0032af6d0aa136764720687e9a05]{
 -- Page 87, Offset 24478.
 -- 感情と目的は別。協力関係って、そういうコトだろ」
-Feelings are separate from objectives. That's what it means to work together, right?"
+We seperate our feelings from our objectives. That's what it means to work together, right?"
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_2_2.txt:177]: separate

File: script/Arcueid/Day 10/10_00_ARC10_2_2.txt [hunk 4]
```diff
@@ -156,11 +158,12 @@ And so...
 [sha:31920d97c6a9151340749b5c7214c356b3849719]{
 -- Page 86, Offset 24475.
 -- だいたい、俺がおまえを嫌いだって言ったらどうする気なんだ？　見逃してくれるのか？」
-Besides, what would you do if I told you that I hated you? Would you have let me go?"
+Besides, what would you do if I had told you that I hated you? Would you have let me go?"
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_2_2.txt:161]: what would you have done if I'd

File: script/Arcueid/Day 10/10_00_ARC10_2_2.txt [hunk 5]
```diff
@@ -186,7 +189,7 @@ Usually Arcueid is a rational thinker.
 [sha:b0926397a62797a3532a899da521f70f98cb920a]{
 -- Page 88, Offset 24481.
 -- そのアルクェイドがあんな質問をしてきたぐらい、昨夜はお怒りだったのだろう。
-If she is going around asking me a question like that, then she must have been really furious last night.
+If she asking me questions like that, then she must have been really furious last night.
```
Comment (yk1512) [inline script/Arcueid/Day 10/10_00_ARC10_2_2.txt:192]: If she's asking

**Review by yk1512 — CHANGES_REQUESTED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_2_1.txt [hunk 6]
```diff
@@ -11,12 +11,12 @@ You're essentially selfish at heart after all."
 [sha:b03d6bd04f2db57407a60160697551f9ecfa7693]{
 -- Page 10002, Offset 24440.
 -- きっぱりと、思ったままの意見を告げた。
-I clearly get the thought on my mind across.
+I express my thinking clearly.
```
Comment (nolo16) [inline script/Arcueid/Day 10/10_00_ARC10_2_1.txt:14]: I express my thoughts clearly.

**Review by nolo16 — APPROVED** [review]

File: script/Arcueid/Day 10/10_00_ARC10_2_2.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 10/10_00_ARC10_2_2.txt:161]: this is actually an italics candidate. Forgot to mark it.

**Pull request merged**